src/networks.json

```diff
@@ -1196,6 +1196,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2400": {
+    "key": "2400",
+    "name": "TCG Verse Mainnet",
+    "shortName": "TCGV",
+    "chainId": 2400,
+    "network": "mainnet",
+    "multicall": "0xceC65DEE0b5012F1b7321b2647681F997c7204FC",
+    "rpc": [
+      "https://rpc.tcgverse.xyz"
```

Comment:
Hi @akira-19 This node has very less payload limit, It should work when we pass atleast 500 addresses into a multicall

Reply:
@ChaituVR Thank you. I'm checking.

Reply:
@ChaituVR It has been fixed.